conf/alluxio-env.sh.template

```diff
@@ -68,3 +71,7 @@
 # Additional classpath entries for Alluxio processes. (Default: "")
 # E.g. "/path/to/library1/:/path/to/library2/"
 # ALLUXIO_CLASSPATH
+
+# Configuring remote debugging for Alluxio master process
```

Comment:
nit: this file enumerates on all the existing non-deprecated env vars such as all the *_JAVA_OPTS for the various processes (note that SECONDARY_MASTER is no longer used but left for backcompat). i would also enumerate all the new ones you've defined in launch-process for completeness.

now that i stared a little longer at this, the hub agent and hub manager ones were not added to this file when they were introduced @tieujason330

Reply:
and also mention the default value to be consistent with all the previous descriptions even if it may seem silly to indicate its an empty string

Reply:
@jenoudet would you be add to add hub agent/manager sections? or would you prefer i do it in a separate pr?

Reply:
Could you do it in a followup PR please? I don't know what a good example of `JAVA_OPTS` would be for the hub agent.